src/main/scala/org/constellation/ConfigEndpoints.scala

```diff
@@ -22,6 +22,10 @@ trait ConfigEndpoints {
       path("checkpointFormation") {
         dao.enableCheckpointFormation()
         complete(StatusCodes.OK)
+      } ~
```

Comment:
IMHO we should slowly transit for creating unnecessary endpoints for each config property, first of all we should use hocon instead of hardcoded keys (every change = compile) there should be also distinction of dynamic configuration that can be modified on runtime (via endpoint) and static provided on startup  of jvm (or read as environmental variable). I am not saying that you should refactor all the places but not making it worst would be good practice.